prototype/src/main/test/java/com/google/sps/test/TestPlace.java

```diff
@@ -0,0 +1,102 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * The package that currently holds all the test files of the food-mood project.
+ * @since 1.0
+ * @author karenav
+ * @version 1.1
+ */
+package com.google.sps.test;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import com.google.sps.data.Place;
+
+@RunWith(JUnit4.class)
+public final class TestPlace {
+
+  /** A stub Place name. */
```

Comment:
thank you for that input. I took all the specific constants inside the functions. 
left the ones that are used in all functions outside and renamed them, because I think that it's better to keep them unified. for example, if we change the rating system and now 3 becomes invalid, I think it makes more sense to change it in one place and not in all functions (there may even be more down the road). what do you think?